Logger.Tests/FileLoggerTests.cs

```diff
@@ -1,10 +1,71 @@
-﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
-
-namespace Logger.Tests
-{
-    [TestClass]
-    public class FileLoggerTests
-    {
-        
-    }
-}
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.IO;
+using System;
+using System.Linq;
+
+namespace Logger.Tests
+{
+    [TestClass]
+    public class FileLoggerTests
+    {
+        [TestMethod]
+        [ExpectedException(typeof(ArgumentNullException))]
+        public void FileLogger_NullFile_ThrowsArgumentNullException()
+        {
+            // Act
+            FileLogger? fileLogger = new FileLogger(null!);

```

Comment:
```suggestion
            FileLogger? fileLogger = new FileLogger(null);
```
`!` can be removed if you set the file name parameter to be nullable.